iree/tools/CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ if(IREE_ENABLE_EMITC)
   )
 endif()
 
+configure_file(config.txt.in config.txt)
```

Comment:
Can you add a brief comment here? (or in the file, but then it would need to make sense in both the template and the generated file)

```
# Write the current CMake options to a file for convenient use from scripts.
```

(more descriptive names would also help)

Reply:
Added.